web-admin/src/features/alerts/metadata/AlertMetadata.svelte

```diff
@@ -75,6 +82,20 @@
   $: emailNotifier = extractNotifier(alertSpec?.notifiers, "email");
   $: slackNotifier = extractNotifier(alertSpec?.notifiers, "slack");
 
+  $: exploreUrl = getMappedExploreUrl(
+    {
+      exploreName: $exploreName.data ?? "",
```

Comment:
It feels wrong to pass an empty string to this function. Could we handle the types better?